### DISCUSSION TITLE
revanced 60fps youtube videos Lag issue

### DISCUSSION THREAD

**Discussion by mustpha152002** [ReVanced/revanced-discussions#1057] [Question] [answered]
Revanced 60fps YouTube videos are very laggy while in the original app it has no lag why is that? I 'am using Samsung galaxy F04 it has 4gb ram when i was using my old phone Samsung galaxy A20 i did not had this issue.

**Comment by LisoUseInAIKyrios** [comment]
Are you using spoof device dimensions?

What version YT did you patch, and what version patches?

Try turning off client spoof or turning off iOS spoofing and see if it changes.

If it still acts up, try uninstalling stock YT and install unpatched 19.16.39 and see if that acts the same.

**Comment by mustpha152002** [comment, marked as answer]
nvm it was the ios spoofing that was causing it thanks for the help
